selfdrive/car/chrysler/carstate.py

```diff
@@ -112,7 +112,7 @@ def update(self, cp, cp_cam):
     self.seatbelt = (cp.vl["SEATBELT_STATUS"]['SEATBELT_DRIVER_UNLATCHED'] == 0)
 
     self.brake_pressed = cp.vl["BRAKE_2"]['BRAKE_PRESSED_2'] == 5 # human-only
-    self.pedal_gas = cp.vl["ACCEL_PEDAL_MSG"]['ACCEL_PEDAL']
```

Comment:
in the dbc that's 4 bits though. Should it be 1?

Reply:
I think 4 bits should be fine, as it's compared with > 0 in interface.py:
`ret.gasPressed = self.CS.pedal_gas > 0`

7 bits for ACCEL_134 is more accurate and I've submitted a PR here: https://github.com/commaai/opendbc/pull/174
Although in practice 4 bits is just fine as the low 4 bits will be set anyways as the user presses on the pedal and its value fluctuates.

Thanks for looking at the PR!

Reply:
ah perfect. Ok, I'll merge this.